### PR TITLE
Css variables - Adding Open Sans font PR

### PR DESCRIPTION
**Pull request by wchrapcynski**
Per issue #137, Open Sans has been added in to App.css.

### CODE REVIEW

**Comment by wchrapcynski** [comment]
Also looks like there were a couple shades of gray missing. Added those into App.css.

File: client/src/App.js [hunk 1]
```diff
@@ -1,24 +1,13 @@
...
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+<<<<<<< HEAD
+=======
+  
+>>>>>>> parent of 08b33aa... Revert "Updated css variables with another font that was needed. Also cleaned up App and removed default React setup code."
     </div>
```
Comment (ZakkMan) [inline client/src/App.js]: Can you fix this piece

**Comment by wchrapcynski** [comment]
@ZakkMan This should be good to go now.

File: client/src/App.js (hunk 1 above)
Comment (wchrapcynski) [reply client/src/App.js]: I accidentally edited in the wrong branch but I should have reverted back. Hopefully all should be good now.

**Pull request merged**